ewmh.c

```diff
@@ -256,14 +259,15 @@ ewmh_update_net_numbers_of_desktop(void)
 			_NET_NUMBER_OF_DESKTOPS, XCB_ATOM_CARDINAL, 32, 1, &count);
 }
 
-void
-ewmh_update_net_current_desktop(void)
+int
+ewmh_update_net_current_desktop(lua_State *L)
```

Comment:
>It does not use lua_State (anymore)?!

Uh, it did not, and does not. Still it should have appropriate signature to be used as signal handler, or so i thought...correct me if i'm wrong.